ext/zend_test/tests/observer_retval_07.phpt

```diff
@@ -0,0 +1,39 @@
+--TEST--
+Observer: Retvals are observable that are: IS_REFERENCE, IS_VAR
+--SKIPIF--
+<?php if (!extension_loaded('zend-test')) die('skip: zend-test extension required'); ?>
+--INI--
+zend_test.observer.enabled=1
+zend_test.observer.observe_all=1
+zend_test.observer.show_return_value=1
+--FILE--
+<?php
+function &getMessage() {
+    $retval = 'I should be observable';
```

Comment:
This needs to be something refcounted to check for use after free.

Reply:
A refcounted retval is being tested in `observer_retval_04.phpt`. In this test I was targeting [this branch](https://heap.space/xref/php-src/Zend/zend_vm_def.h?r=855d8fa6#4292-4300). Do you think we have sufficient coverage with all the retval tests?